framer/Utils.coffee

```diff
@@ -76,6 +77,12 @@ Utils.nearestIncrement = (x, increment) ->
 window.requestAnimationFrame ?= window.webkitRequestAnimationFrame
 window.requestAnimationFrame ?= (f) -> Utils.delay 1/60, f
 
+Utils.animatorFor = (options) ->
```

Comment:
Does this need to be exposed? Seems like an internal thing mostly, maybe mark it as such in some way?